### ISSUE TITLE
Train/test split

### ISSUE THREAD

**Issue by nicolaiberk** [nicolaiberk/nlpdl_project#1]
In the baseline model, split off a test set consisting of press releases published _after_ the training-validation set.
